contrib/jemalloc-cmake/CMakeLists.txt

```diff
@@ -170,7 +170,7 @@ endif ()
 target_compile_definitions(_jemalloc PRIVATE -DJEMALLOC_PROF=1)
 
 if (USE_UNWIND)
-    target_compile_definitions (_jemalloc PRIVATE -DJEMALLOC_PROF_LIBUNWIND=1)
+    target_compile_definitions (_jemalloc PRIVATE -DJEMALLOC_PROF_LIBGCC=1)
```

Comment:
Actually `unw_backtrace` is used not only in jemalloc, but also in StackTrace.cpp (added in f8d2b5effd0c177f605ce9d27b640753f601ecb6):

https://github.com/ClickHouse/ClickHouse/blob/6d3a54a0440050e531f99976282e6195972a63bb/src/Common/StackTrace.cpp#L305

So this change looks redundant
But it worth to update a comment for `unw_backtrace`

https://github.com/ClickHouse/libunwind/blob/c4ea9848a697747dfa35325af9b3452f30841685/src/libunwind.cpp#L287

Reply:
Indeed, StackTrace.cpp uses unw_backtrace directly.
I think that this usage should be replaced with _Unwind_Backtrace too.
I might proprose a patch later on.

Multiple crutches regarding unw_backtrace() implementation could be removed after this replacement.